app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -39,7 +47,16 @@ keywords:
 
 If you’re applying for teacher training as a non-UK citizen, you'll need a [visa or immigration status allowing you to study or work in England](#check-your-status). 
 
-Most candidates who don't already have a visa or immigration status allowing them to study or work in England will need to apply for a Student or Skilled Worker visa.
+Most candidates who do not already have a visa or immigration status allowing them to study or work in England will need to apply for a Student or Skilled Worker visa.
+<section class="inset-text purple">
```

Comment:
Should this be defined in the front matter like the other inset text on the page?